tests/README.md

```diff
@@ -34,6 +34,12 @@ To run tests without using run_tests script:
 
     `$ unit2 discover core/tests`
 
+
+Compile sources with Python 3
+-----------------------------
+To make sure the tank source code can be compiled with Python 3, run the compile_python3.* scripts. Note that the unit tests cannot
+be executed right now with Python 3.
+
```

Comment:
Noticed the readme further down is a bit weird - maybe this is a good time to adjust?

![image](https://user-images.githubusercontent.com/337710/31665310-4af67076-b340-11e7-98cf-9e1b7c87374d.png)

Reply:
We also recommend importing `*` which i think we probably want to move away from:

![image](https://user-images.githubusercontent.com/337710/31665332-5ea4fe9e-b340-11e7-946d-f09a86e51bdf.png)

Maybe you could do a quick pass just to double check the readme looks up to date with what we typically do today test wise?